astropy/utils/masked/core.py

```diff
@@ -1026,7 +1026,7 @@ def argmax(self, axis=None, out=None, *, keepdims=False):
         at_max = self == self.max(axis=axis, keepdims=True)
         return at_max.filled(False).argmax(axis=axis, out=out, keepdims=keepdims)
 
-    def argsort(self, axis=-1, kind=None, order=None):
+    def argsort(self, axis=-1, kind=None, order=None, *, stable=None):
```

Comment:
Unrelated to your PR, but how about writing `kind=np._NoValue, order=None, *, stable=np._NoValue` to indicate even more that they are not used? (for `sort` too)

Reply:
I have a slight preference for sticking with the wrapped function's (user-visible) signature as strictly as possible. I feel this should be discussed as a separate issue, maybe ?